src/pybamm/experiment/step/base_step.py

```diff
@@ -69,6 +76,10 @@ def __init__(
         description=None,
         direction=None,
```

Comment:
This would work with `from __future__ import annotations` at the top.
```suggestion
        direction: Direction | None = None,
```